autopush/main.py

```diff
@@ -532,6 +533,9 @@ def connection_main(sysargs=None, use_files=True):
     resource = DefaultResource(WebSocketResource(factory))
     resource.putChild("status", StatusResource())
     site_factory = Site(resource)
+    # Silence starting/stopping messages
+    site_factory.noisy = args.debug
+    site.noisy = args.debug
```

Comment:
Looks good but let's also set noisy on the other 2 cyclone.web.Applications in this file as well